lib/Twig/SimpleFunction.php

```diff
@@ -87,4 +89,14 @@ public function isVariadic()
     {
         return $this->options['is_variadic'];
     }
+
+    public function isDeprecated()
+    {
+        return $this->options['deprecated'];
+    }
+
+    public function getAlternative()
```

Comment:
I've of course also added some safeguard when calling the `isDeprecated()` method to avoid those fatal errors.